base_layer/wallet_ffi/src/lib.rs

```diff
@@ -2867,10 +2869,19 @@ pub unsafe extern "C" fn wallet_import_utxo(
         return 0;
     }
 
+    let mut message_string = CString::new("").unwrap().to_str().unwrap().to_owned();
+    if !message.is_null() {
+        message_string = CStr::from_ptr(message).to_str().unwrap().to_owned();
+    } else {
+        error = LibWalletError::from(InterfaceError::NullError("message".to_string())).code;
+        ptr::swap(error_out, &mut error as *mut c_int);
```

Comment:
Is the intended behaviour to ignore a empty message and continue?

Reply:
Good point, let me put a default in there